sdk/monitor/monitor-query/review/monitor-query.api.md

```diff
@@ -81,28 +81,28 @@ export interface GetMetricNamespacesResponse {
 }
 
 // @public
-export interface LocalizableString {
-    localizedValue?: string;
-    value: string;
-}
-
-// @public
-export class LogsClient {
-    constructor(tokenCredential: TokenCredential, options?: LogsClientOptions);
-    // (undocumented)
-    queryLogs(workspaceId: string, query: string, options?: QueryLogsOptions): Promise<QueryLogsResult>;
+export class LogsQueryClient {
+    constructor(tokenCredential: TokenCredential, options?: LogsQueryClientOptions);
+    queryLogs(workspaceId: string, query: string, timespan: string, options?: QueryLogsOptions): Promise<QueryLogsResult>;
     // (undocumented)
     queryLogsBatch(batch: QueryLogsBatch, options?: QueryLogsBatchOptions): Promise<QueryLogsBatchResponse>;
 }
 
 // @public (undocumented)
-export interface LogsClientOptions extends PipelineOptions {
+export interface LogsQueryClientOptions extends PipelineOptions {
     endpoint?: string;
 }
 
+// @public
+export interface LogsTable {
+    columns: MetricColumn[];
+    name: string;
+    rows: (Date | string | number | Record<string, unknown> | boolean)[][];
```

Comment:
Should we give this type a name? 

```ts
rows: MetricRow[][]
```

Reply:
Yes.

Also, the 'MetricColumn' type above is wrong, I believe. I need to look into how that rename propagated like that.

Reply:
Thinking about this a bit more. MetricRow would be wrong since that's just the type of an individual "cell".

Let me see what everyone else did here.

Reply:
Derp, yes that would be `MetricRow[]` where `MetricRow` is a type `MetricCell[]` or something along those lines (depending on what others are doing) 😄 one too many `[]` in my original comment

Reply:
Okay, LogsColumn is in.

I'm going to leave the LogsRow/MetricsRow thing out for now. In other languages they have to wrap the underlying type because it isn't just a simple array, like ours. 

I was thinking about wrapping the various types, however but there is this prior art in Service Bus, where we have a field that can be a variety of different types and we didn't give it a name.

https://github.com/Azure/azure-sdk-for-js/blob/486ca8a08891d9fa7d49708ba77c9a05dcb2716a/sdk/servicebus/service-bus/src/serviceBusMessage.ts#L213